scripts/db/schema.sql

```diff
@@ -0,0 +1,168 @@
+ * INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF
+ * THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF REGENTS HAS BEEN ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * REGENTS SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
+ * SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, PROVIDED HEREUNDER IS PROVIDED
+ * "AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ * ENHANCEMENTS, OR MODIFICATIONS.
+ */
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+SET search_path = public, pg_catalog;
+SET default_tablespace = '';
+SET default_with_oids = false;
+
+CREATE TABLE canvas_site_mailing_list_members (
+    id SERIAL PRIMARY KEY,
+    mailing_list_id INTEGER NOT NULL,
+    email_address CHARACTER VARYING(255) NOT NULL,
+    can_send BOOLEAN DEFAULT FALSE NOT NULL,
+    first_name CHARACTER VARYING(255),
+    last_name CHARACTER VARYING(255),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    deleted_at TIMESTAMP WITH TIME ZONE,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    welcomed_at TIMESTAMP WITH TIME ZONE
+);
+ALTER TABLE ONLY canvas_site_mailing_list_members
+    ADD CONSTRAINT canvas_site_mailing_list_members_unique_constraint UNIQUE (mailing_list_id, email_address);
+CREATE INDEX canvas_site_mailing_list_members_deleted_at_idx
+    ON canvas_site_mailing_list_members USING btree (deleted_at);
+CREATE INDEX canvas_site_mailing_list_members_welcomed_at_idx
+    ON canvas_site_mailing_list_members USING btree (welcomed_at);
+
+--
+
+CREATE TABLE canvas_site_mailing_lists (
+    id SERIAL PRIMARY KEY,
+    canvas_site_id INTEGER NOT NULL,
+    canvas_site_name CHARACTER VARYING(255),
+    list_name CHARACTER VARYING(255),
+    members_count INTEGER,
+    populate_add_errors INTEGER,
+    populate_remove_errors INTEGER,
+    populated_at TIMESTAMP WITH TIME ZONE,
+    state CHARACTER VARYING(255),
+    type CHARACTER VARYING(255),
+    welcome_email_active BOOLEAN DEFAULT FALSE NOT NULL,
+    welcome_email_body TEXT,
+    welcome_email_subject TEXT,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+ALTER TABLE ONLY canvas_site_mailing_lists
+    ADD CONSTRAINT canvas_site_mailing_lists_unique_constraint UNIQUE (canvas_site_id);
+
+--
+
+CREATE TABLE canvas_synchronization (
+    id SERIAL PRIMARY KEY,
+    last_guest_user_sync TIMESTAMP WITH TIME ZONE,
+    latest_term_enrollment_csv_set TIMESTAMP WITH TIME ZONE,
+    last_enrollment_sync TIMESTAMP WITH TIME ZONE,
+    last_instructor_sync TIMESTAMP WITH TIME ZONE
+);
+
+--
+
+CREATE TABLE oauth2_data (
```

Comment:
@johncrossman I'm pretty sure we can get by without this table as well. At this point Junction only uses it to upload CSVs to bDrive and we're going to get Ripley out of that business.